composer.json

```diff
@@ -15,6 +15,13 @@
             "EzSystems\\EzPlatformMatrixFieldtype\\": "src/lib/"
         }
     },
+    "autoload-dev": {
+      "psr-4": {
+        "EzSystems\\EzPlatformMatrixFieldtype\\Integration\\Tests\\": "tests/integration/"
+      }
+    },
+    "minimum-stability": "dev",
```

Comment:
Shouldn't be needed here (1.0), right? And for master (if it fails to install), we should add the missing packages to require-dev section (and not use minimum-stability, prefer-stable flags).

Reply:
Fixed in https://github.com/ezsystems/ezplatform-matrix-fieldtype/pull/25/commits/7dc3dda4c0ded4abf40ae4da7f483b80726b3455